static/htk/js/toolbar/emulate_user.js

```diff
@@ -0,0 +1,107 @@
+$(function() {
+    // Toolbar Elements
+    var htkToolbar = $('.htk-toolbar');
+    var htkToolbarHideButton = $('.htk-toolbar-hide');
+    var htkToolbarTab = $('.htk-toolbar-tab');
+
+    // Toolbar Constants
+    var EMULATE_USER_ID = 'emulate_user_id';
+    var EMULATE_USERNAME = 'emulate_user_username';
+    var HTK_EMULATE_USER_COOKIE_EXPIRE_TIME_SECONDS = 15 * 60 * 1000 // 15 minutes
```

Comment:
not really, but, meh whatever, okay

Reply:
weren't we going to put this in `constants` and read it in somewhere using `htk_setting`?

Reply:
Ya, I was working on the css part for a long time today and haven't gotten to this part yet.